Project.toml

```diff
@@ -4,10 +4,13 @@ authors = ["Jérémie Gillet <jie.gillet@gmail.com> and contributors"]
 version = "1.3.1"
 
 [deps]
+ColorTypes = "3da002f7-5984-5a60-b8a6-cbb66c0b333f"
```

Comment:
Seems not used?

Reply:
I also didn't see anything in the diffs that contains `using ColorTypes` or `import ColorTypes`